pkg/background/cleanup_jobs.go

```diff
@@ -0,0 +1,97 @@
+package background
+
+import (
+	"context"
+	"time"
+
+	"github.com/getsentry/sentry-go"
+	"github.com/go-pg/pg/v10"
+	"github.com/monetr/monetr/pkg/models"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	CleanupJobs = "CleanupJobs"
```

Comment:
Right now with the current changes, nothing is ever inserted into the jobs table. With that this job could either be disabled for the time being. Or changes could be made to support the jobs table.